gravityflow/gravityflowblocks
# Add the Reports block

This PR adds the Reports block to the Gutenberg editor. Currently it requires [this branch of GravityFlow](https://github.com/gravityflow/gravityflow/tree/update-reports-filter) to work.

## Testing instructions
1. Add a Reports block.
2. Update the block attributes in the sidebar and check if the charts updated correctly in the block.
3. Check the frontend and see if the shortcode rendered correctly.
4. Adding multiple Reports blocks and see if them work in the block editor and frontend.
